besthouse/client/main.go

```
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/mlgx/puzzling/besthouse"
+)
+
+func mapGenerate(mReq besthouse.MapReq) (*besthouse.MapRes, error) {
+	buf, err := json.Marshal(mReq)
+	if err != nil {
+		return nil, err
+	}
+	body := bytes.NewReader(buf)
+	req, err := http.NewRequest("GET", "http://127.0.0.1:8080/generate", body)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := http.DefaultClient.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	if resp.StatusCode != http.StatusOK {
+		s, _ := ioutil.ReadAll((resp.Body))
```

By convention, when discarding an error, we need to comment on why. (simple inlined comment `// Best effort.`  would do the trick here)